cloudbuild.yaml

```diff
@@ -46,13 +55,16 @@ steps:
       - '-c'
       - |
         set -e
-        terraform apply -var-file=tfvars_tests/standard-gke-public.platform.tfvars -var=project_id=$PROJECT_ID \
+        terraform apply \
+        -var-file=tfvars_tests/standard-gke-public.platform.tfvars \
+        -var=project_id=$PROJECT_ID \
         -var=cluster_name=ml-$SHORT_SHA-$_PR_NUMBER-cluster \
-        -var=cluster_region=$_REGION -auto-approve -no-color
+        -var=cluster_region=$_REGION \
+        -auto-approve -no-color -lock=false
```

Comment:
Why do we need `-lock=false`?

Reply:
During some of my TF runs, failed TF apply didn't properly release the tf lock hence the cleanup step failed to execute. With lock=false, cleanup steps run without getting blocked.